alfresco-solrapi-client/alfresco-solrapi-client-api/src/main/java/eu/xenit/alfresco/client/solrapi/api/model/SolrNodeMetaData.java

```diff
@@ -1,37 +1,34 @@
 package eu.xenit.alfresco.client.solrapi.api.model;
 
-import java.util.ArrayList;
-import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import lombok.Data;
+import lombok.Value;
 import lombok.experimental.Accessors;
 
-@Data
+@Value
```

Comment:
This makes the class final

It is no longer possible to inherit from this class, like for example in an ExtendedSolrNodeMetadata class

Reply:
Or should we subclass the SolrNodeMetadataModel for every implementation from now on?